examples/text-generation/run_generation.py

```diff
@@ -245,6 +253,10 @@ def main():
     args = setup_parser(parser)
     model, tokenizer, generation_config = initialize_model(args, logger)
 
+    use_lazy_mode = True
+    if args.torch_compile:
```

Comment:
We could avoid this by using args.torch_compile directly in 312 and 492 with lazy_mode = not args.torch_compile.

Reply:
Yes, that can be done of course. But, I wanted to have the decision making about use_lazy_mode in one place for all. A secondary reason is that I plan to include additional checks in future PR to control use_lazy_mode.